### PR TITLE
Add plugin dependencies help tab on plugins pages

### PR DESCRIPTION
**Pull request by DorZuberi**
Trac ticket: https://core.trac.wordpress.org/ticket/60466

**Preview:**
image

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/plugins.php [hunk 1]
```diff
@@ -602,6 +602,19 @@
...
 }
 
+if ( current_user_can( 'install_plugins' ) ) {
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'plugins-dependencies',
+			'title'   => __( 'Dependencies' ),
+			'content' =>
+				'<p>' . __( 'Plugin Dependencies aims to make the process of installing and activating addons (dependents) and the plugins they rely on (dependencies) consistent and easy.' ) . '</p>' .
+				'<p>' . __( 'If a dependency plugin is deleted via FTP or deployment, a notice will be displayed on the administration’s plugin screens, informing the user that there are missing dependencies to install and/or activate. Additionally, each dependent whose dependencies are no longer met will have an error notice in their plugin row.' ) . '</p>' .
```
Comment (audrasjb) [inline src/wp-admin/plugins.php]: > on the administration’s plugin screens
it sounds a bit unclear to me… maybe simplify with "on the Plugin administration screens" ?

File: src/wp-admin/plugins.php [hunk 2]
```diff
@@ -602,6 +602,19 @@
...
+				'<p>' . __( 'In case plugin activation link is disabled, this is due to missing required dependencies for the said plugin.' ) . '</p>',
```
Comment (audrasjb) [inline src/wp-admin/plugins.php]: Maybe this way: "If a dependent plugin is missing some dependencies, its activation button is disabled until the related dependencies are activated" 
?

**Review by audrasjb — COMMENTED** [review]
Thanks, the patch (code) looks good to me. I left some change proposals though.

File: src/wp-admin/plugins.php (hunk 1 above)
Comment (DorZuberi) [reply src/wp-admin/plugins.php]: Fixed.

File: src/wp-admin/plugins.php (hunk 2 above)
Comment (DorZuberi) [reply src/wp-admin/plugins.php]: Fixed!

File: src/wp-admin/plugins.php [hunk 3]
```diff
@@ -602,6 +602,19 @@
...
 }
 
+if ( current_user_can( 'install_plugins' ) ) {
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'plugins-dependencies',
+			'title'   => __( 'Dependencies' ),
+			'content' =>
+				'<p>' . __( 'Plugin Dependencies aims to make the process of installing and activating addons (dependents) and the plugins they rely on (dependencies) consistent and easy.' ) . '</p>' .
+				'<p>' . __( 'If a dependency plugin is deleted via FTP or deployment, a notice will be displayed on the Plugin administration screens, informing the user that there are missing dependencies to install and/or activate. Additionally, each dependent whose dependencies are no longer met will have an error notice in their plugin row.' ) . '</p>' .
```
Comment (whatcodehaveyougit) [inline src/wp-admin/plugins.php]: I would suggest updating the text to this: 

```suggestion
				'<p>' . __( 'If a required plugin is deleted, a notice will be displayed on the Plugin administration screen informing the user that there is a missing dependency(s) to install and/or activate. Additionally, each plugin whose dependencies are not met will have an error notice on their plugin row.' ) . '</p>' .
```

I would say that this is clearer :)

File: src/wp-admin/plugins.php [hunk 4]
```diff
@@ -602,6 +602,19 @@
...
+				'<p>' . __( 'If a dependent plugin is missing some dependencies, its activation button is disabled until the related dependencies are activated.' ) . '</p>',
```
Comment (whatcodehaveyougit) [inline src/wp-admin/plugins.php]:
```suggestion
				'<p>' . __( 'If a parent plugin is missing a dependency(s), its activation button will be disabled until the required dependency(s) are activated.' ) . '</p>',
```

Slight naming changes, grammer tweak and option of it being plural.

**Review by audrasjb — APPROVED** [review]

**Comment by audrasjb** [comment]
Thanks everyone! Committed in https://core.trac.wordpress.org/changeset/59094

**Pull request closed without merging**